addons/mail/static/src/owl/components/file_uploader/file_uploader.js

```diff
@@ -176,7 +176,9 @@ class FileUploader extends Component {
 }
 
 FileUploader.defaultProps = {
-    fileUploadId: _.uniqueId('o_FileUploader_fileupload')
+    fileUploadId: _.uniqueId('o_FileUploader_fileupload'),
```

Comment:
Know that I think of it... maybe this doesn't work the intended way:
`_.uniqueId()` is computed once, so default will always be the same.

Reply:
This is already merged in master-mail-owl and it's not the purpose of this branch normally. Wouldn't it be better to propose a change in another branch if it needs to be changed ?

Reply:
Agree

Reply:
I created a ticket for that : https://www.odoo.com/web#id=2166383&action=3206&active_id=1519&model=project.task&view_type=form&menu_id=4720